cmd/setup-etcd-environment/run.go

```diff
@@ -164,6 +166,21 @@ func runRunCmd(cmd *cobra.Command, args []string) error {
 			exportEnv["INITIAL_CLUSTER_STATE"] = "existing"
 			return true, nil
 		})
+
+		ep, err := client.CoreV1().Endpoints("openshift-etcd").Get("etcd", metav1.GetOptions{})
+		if err != nil {
+			return err
+		}
+		if len(ep.Subsets) >= 1 {
+			if len(ep.Subsets[0].Addresses) == 0 {
```

Comment:
Would this fail in a single-member restore?

Reply:
No etcd exists we restore to single-member from snapshot for example, this doesn't assume bootstrap when we add another member does it?